appscale/tools/local_state.py

```diff
@@ -104,7 +104,7 @@ def ensure_appscale_isnt_running(cls, keyname, force):
 
       acc = AppControllerClient(login_host, secret_key)
       try:
-        acc.get_status()
+        acc.get_all_public_ips()
```

Comment:
Got it, makes sense. If you could pick something more innocuos would be best (I think this function should be removed to favored the nodes layout we have for example), maybe is_done_loading, or or get_role_info etc...